index.html

```diff
@@ -26,20 +22,39 @@
       <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
-
 </head>
 
-  <body>
-
-    <div class="posts"></div>
-
-
+  <body class ="lista">
+  <!-- Start Bar -->
+  <p class="cblog">Cognits Blog</p>
```

Comment:
Indentation and following good coding practices helps your code be much more readable by yourself and other coders. This is a very important habit to have because it makes your code much more maintainable in the future, check out this Wikipedia article: http://en.wikipedia.org/wiki/Programming_style.